vite.config.mts

```diff
@@ -34,6 +34,10 @@ export default defineConfig(({ _, mode }) => {
                         isCustomElement: (tag) => tag === 'cesium-compass',
                     },
                 },
+                script: {
+                    // TODO remove this once updated vue to 3.4
```

Comment:
did you try to update, with all the minor update that have been published lately? maybe the issue we saw back a week ago is somehow fixed now